sferes/stat/pareto_samples.hpp

```diff
@@ -0,0 +1,103 @@
+//| This file is a part of the sferes2 framework.
```

Comment:
I would prefer to have the content of this file directly in ex_nsga2 (to make the example more self-contained).